lib/tests/test_logthrdestdrv.c

```diff
@@ -776,6 +777,7 @@ static void
 setup(void)
 {
   app_startup();
+  init_and_load_syslogformat_module();
```

Comment:
Instead of using an unneeded dependency (`syslogformat`) wouldn't it be possible to create a new logmsg, I mean this test has nothing to do with `syslogformat`

Reply:
Good point. But for now I just simply drop that patch because it is a cosmetic change.